street_network/src/lanes/classic.rs

```diff
@@ -31,33 +34,20 @@ pub fn get_lane_specs_ltr(tags: &Tags, cfg: &MapConfig) -> Vec<LaneSpec> {
     if tags.is_any("railway", vec!["light_rail", "rail"]) {
         return vec![fwd(LaneType::LightRail)];
     }
-    if tags.is(osm::HIGHWAY, "steps") {
-        return vec![fwd(LaneType::Sidewalk)];
-    }
-    // Eventually, we should have some kind of special LaneType for shared walking/cycling paths of
-    // different kinds. Until then, model by making bike lanes and a shoulder for walking.
-    if tags.is_any(
-        osm::HIGHWAY,
-        vec!["cycleway", "footway", "path", "pedestrian", "track"],
-    ) {
-        // If it just allows foot traffic, simply make it a sidewalk. For most of the above highway
-        // types, assume bikes are allowed, except for footways, where they must be explicitly
-        // allowed.
-        if tags.is("bicycle", "no")
-            || (tags.is(osm::HIGHWAY, "footway")
-                && !tags.is_any("bicycle", vec!["designated", "yes", "dismount"]))
-        {
-            return vec![fwd(LaneType::Sidewalk)];
-        }
-        // Otherwise, there'll always be a bike lane.
 
+    // If it's a primarily cycleway, have directional bike lanes and add a shoulder for walking
```

Comment:
It's still odd to use two directional lanes for this case. I think `street_network::Direction` and everything downstream needs to have a both-ways case. It's a very complex change downstream, affecting routing, rendering, traffic simulation... so not attempting it yet.

Reply:
Yes, I agree. An "alternates" case too.

Most reads of the value will boil down to "is travel supported in this direction", right? Routing could implement "alternating" and use that for "both ways" too.

I called it [`TravelDirection` in my types](https://github.com/a-b-street/osm2streets/blob/a19ab6ac8ef1dc4b812830fc1eae11687e28356c/osm2streets/src/units.rs#L22-L33).